Makefile

```diff
@@ -183,14 +188,15 @@ misspell-fix: $(TOOLBIN)/misspell
 ## --------------------------------------
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+# This is expected to be used by user during dev
 .PHONY: deploy
-deploy: $(TOOLBIN)/kubectl
-	$(TOOLBIN)/kubectl apply -f deploy/kube-app-manager-aio_$(VERSION).yaml
+deploy: $(TOOLBIN)/kubectl generate-resources
```

Comment:
what about a double target ? 
```
deploy-dev
deploy
```
I'd prefer to skip build dependencies and default to the 'release' file for the `make deploy`

Reply:
ack. will fix it.

Reply:
Done. Separate make targets added.